### DISCUSSION TITLE
Log Rotation for wait_for_web_response.log

### DISCUSSION THREAD

**Discussion by Masaya-A** [Security-Onion-Solutions/securityonion#4239] [Unsupported Versions] [answered]
Since 2.3.40, `/root/wait_for_web_response.log` is growing and is approx. 700MB for my rigs today.
Could you add it to the log rotation, please?

**Comment by jertel** [comment]
Tracking in #4247.

**Comment by TOoSmOotH** [comment, marked as answer]
2.3.51 Should address this. Let me know if it doesn't.
